tools/topology/topology2/include/dais/hda.conf

```diff
@@ -0,0 +1,85 @@
+#
+# HDA DAI
+#
+# All attributes defined herein are namespaced by alsatplg to "Object.Dai.hda.attribute_name"
+#
+# Usage: this component can be used by declaring as follows:
+#
+# For Playback
+#Object.HDA."N" {
```

Comment:
@RanderWang can you please fix the comments here? to show the right usage? It should be Object.Dai.HDA.N

Reply:
sure